<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>笔记本</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_open">
          <property name="text">
           <string>open</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="textEdit_1"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_tansform">
            <property name="text">
             <string>转义</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>打开文件路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label2">
            <property name="text">
             <string>保存文件路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelGif">
          <property name="text">
           <string>GIF</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件（&amp;F）</string>
    </property>
    <addaction name="actionZin"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose_C"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionSave_as_L"/>
    <addaction name="actionExit_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑（&amp;E）</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="action_X"/>
    <addaction name="action_C"/>
    <addaction name="action_V"/>
    <addaction name="action_F"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助（H）</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="statusTip">
    <string>新建</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZin"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionClose_C"/>
   <addaction name="separator"/>
   <addaction name="actionSave_S"/>
   <addaction name="separator"/>
   <addaction name="actionSave_as_L"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionZin">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/image/new.jpg</normaloff>:/myImages/image/new.jpg</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="toolTip">
    <string>新建(N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/image/open_file.jpg</normaloff>:/myImages/image/open_file.jpg</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>打开</string>
   </property>
  </action>
  <action name="actionClose_C">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/image/colse.jpg</normaloff>:/myImages/image/colse.jpg</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="statusTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/image/save.jpg</normaloff>:/myImages/image/save.jpg</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="statusTip">
    <string>存为</string>
   </property>
  </action>
  <action name="actionSave_as_L">
   <property name="icon">
    <iconset resource="myResources.qrc">
     <normaloff>:/myImages/image/save_.jpg</normaloff>:/myImages/image/save_.jpg</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;K)</string>
   </property>
   <property name="statusTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionExit_X">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>版本说明</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myResources.qrc"/>
 </resources>
 <connections/>
</ui>
